soc/nxp/imx/imx8m/Kconfig.soc

```diff
@@ -40,7 +40,7 @@ config SOC_MIMX8ML8_ADSP
 	help
 	  Enable support for NXP i.MX 8MPLUS Audio DSP
 
-config SOC_MIMX8MP_M7
```

Comment:
@iuliana-prodan I would also add some comments saying 

```# i.MX8MP in human speak```

Reply:
There are some info in the help, see, for example:
```
help
	  Enable support for NXP i.MX 8MPLUS M7 MCU
```
or
```
help
	  Enable support for NXP i.MX 8MPLUS Audio DSP
```

Reply:
thanks.this will do.